### ISSUE TITLE
encode error with cpp

### ISSUE THREAD

**Issue by Svtter** [maralla/completor.vim#89]
When I use completor.vim, I got an error like this:

```
"main.cpp" 87L, 1927C
Error detected while processing function <SNR>109_complete[4]..completor#utils#get_completer:
line    1:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/svtter/.vim/plugged/completor.vim/pythonx/completor/api.py", line 14, in wrapper
    return func(vim.bindeval('a:'))
  File "/Users/svtter/.vim/plugged/completor.vim/pythonx/completor/api.py", line 22, in get_completer
    return [c.format_cmd(), c.filetype, c.daemon, c.sync] if c else []
  File "/Users/svtter/.vim/plugged/completor.vim/pythonx/completers/cpp.py", line 44, in format_cmd
    '-I{}'.format(self.current_directory),
UnicodeEncodeError: 'ascii' codec can't encode characters in position 31-35: ordinal not in range(128)
line    2:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'res' is not defined
E858: Eval did not return a valid python object
E471: Argument required
```

and here's my source file:
https://gist.github.com/Svtter/e5b338e448a0c843547d9e3dfb8dd81f
<img width="850" alt="wx20170405-111330 2x" src="https://cloud.githubusercontent.com/assets/5856931/24688217/1a74687e-19f1-11e7-8bb8-fdda9733f6d6.png">

**Comment by maralla** [comment]
Thanks for reporting. This fix #90 should resolve this problem.

**Comment by Svtter** [comment]
@maralla fixed. Thanks a lot.
